src/inspector/models/popups/harppedal/harppedalpopupmodel.h

```diff
@@ -86,7 +86,7 @@ public slots:
 } //namespace mu::notation
```

Comment:
This comment is now no longer accurate (let's remove it)

Reply:
Should I also change the ifndef and define statements? any why don't we use `#pragma once` in musescore?